packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -1034,16 +1074,30 @@ export const createAzureCommunicationCallWithChatAdapter = async ({
   });
 
   const chatThreadAdapter = _createChatThreadAdapterInner(locator, callAdapter);
-  const chatAdapter = _createLazyAzureCommunicationChatAdapterInner(
-    endpoint,
-    userId,
-    displayName,
-    credential,
-    chatThreadAdapter.getChatThread(),
-    'CallWithChat' as _TelemetryImplementationHint
-  );
+  if (chatThreadAdapter.isCallInfoRequired()) {
+    const CCAdapter = new AzureCommunicationCallWithChatAdapter(await callAdapter);
```

Comment:
nit: this feels like an abbreviated name, can we spell it out so that we know what this is in the future?